recipes/djinni-support-lib/all/conanfile.py

```diff
@@ -28,14 +28,14 @@ class DjinniSuppotLib(ConanFile):
     @property
     def objc_support(self):
         if self.options.target == "auto":
-            return self.settings.os in ["iOS", "Macos"]
+            return self.settings.os in ["iOS", "Macos", "watchOS", "tvOS"]
```

Comment:
Couldnt it be this?

https://github.com/conan-io/conan-center-index/blob/340c9d8857c3968c0ab4f732d0f88370d6646ae7/recipes/libgit2/all/conanfile.py#L47

Reply:
No, because macOS is used to create java libraries for macOS

Reply:
Sorry I meant the tools helper not the line of code....

https://docs.conan.io/en/latest/reference/tools.html#tools-is-apple-os

I think this should be changed